src/Model/Table/FeedbackstoreTable.php

```diff
@@ -154,7 +154,8 @@ public function mail(array $feedbackObject, $copyreporter = false) {
 		}
 
 		//Tmp store the screenshot:
-		$tmpfile = APP . 'tmp' . DS . time() . '_' . mt_rand(1000, 9999) . '.png';
+		$tmpFolder = Configure::read('Feedback.configuration.Filesystem.location', ROOT . DS . 'tmp');
+		$tmpfile = $tmpFolder . DS . 'screenshots' . DS . time() . '_' . mt_rand(1000, 9999) . '.png';
```

Comment:
I am not sure if the config value should be used for tmp files though.

Reply:
Updated ! I remove the config value.